package.json

```
@@ -11,6 +11,7 @@
     "now-start": "PRIVATE_KEY=$(echo $PRIVATE_KEY_BASE64 | base64 -d) npm start",
     "test": "jest",
     "test:watch": "jest --watch --notify --notifyMode=change --coverage",
+    "schema": "node ./generate-schema.js print",
```

A bit longer to type, but more clear I think :)

```suggestion
    "generate-schema": "node ./generate-schema.js print",
```